docs/source/metrics.rst

```diff
@@ -381,6 +381,56 @@ Threshold Brier Score
 
 .. autofunction:: _threshold_brier_score
 
+Ranked Probability Score
+========================
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['rps']}")
+
+.. autofunction:: _rps
+
+Reliability
+===========
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['reliability']}")
+
+.. autofunction:: _reliability
+
+Discrimination
+==============
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['discrimination']}")
+
+.. autofunction:: _discrimination
+
+Rank Histogram
+==============
+
+.. ipython:: python
+
+    # Enter any of the below keywords in ``metric=...`` for the compute functions.
+    print(f"\n\nKeywords: {metric_aliases['rank_histogram']}")
+
+.. autofunction:: _rank_histogram
+
+*************************
+Contingency-based metrics
+*************************
+
+All metric scores from ``xskillscore`` are accessible via
+``.verify(metric='contingency', score='hit_rate')``.
```

Comment:
Might want to revise this. It's a bit unclear to me. Maybe:

A number of metrics can be derived from a contingency table (see https://www.cawcr.gov.au/projects/verification/#Contingency_table). To use this in ``climpred``, run ``.verify(metric='contingency', score=...)`` where ``score`` can be any of the following: 

* hits
* misses
* false_alarms
* correct_negatives
* bias_score
* hit_rate
* false_alarm_ratio
* false_alarm_rate
* success_ratio
* threat_score
* equit_threat_score
* odds_ratio
* odds_ratio_skill_score

Reply:
I wouldnt like to name them here (bc there are easily more to be added in xs) but just like to xs docs